web/src/stream_bar.ts

```diff
@@ -4,7 +4,7 @@ import * as stream_data from "./stream_data";
 // color look like the stream being used.
 // (In particular, if there's a color associated with it,
 //  have that color be reflected here too.)
-export function decorate(stream_id, $element) {
+export function decorate(stream_id: number, $element: JQuery): void {
```

Comment:
Experience (#26817) tells me `stream_id` is not always a `number` here. You may have changed which non-`number` is possible in #26866 but I’m pretty sure it’s still possible.

Reply:
Oh oops I didn't realize you already had a PR open. Should I close this one in favor of that?

Reply:
For some reason in that PR I removed the code that handled `undefined` https://github.com/zulip/zulip/pull/26866/files#diff-b14a21b1b800fb57ebc58999afdebcdb1d8f47ea79113303f581b54cd7834738L8 though I don't feel confident about that now